ios/FluentUI/Navigation/Shy Header/ShyHeaderView.swift

```diff
@@ -55,16 +55,16 @@ class ShyHeaderView: UIView, TokenizedControlInternal {
     }
 
     private struct Constants {
-        static let contentHorizontalPadding: CGFloat = 16
-        static let contentTopPadding: CGFloat = 6
-        static let contentTopPaddingCompact: CGFloat = 10
-        static let contentTopPaddingCompactForLargePhone: CGFloat = 0
-        static let contentBottomPadding: CGFloat = 10
-        static let contentBottomPaddingCompact: CGFloat = 6
+        static let contentHorizontalPadding: CGFloat = GlobalTokens.spacing(.size160)
+        static let contentTopPadding: CGFloat = GlobalTokens.spacing(.size80)
+        static let contentTopPaddingCompact: CGFloat = GlobalTokens.spacing(.size80)
+        static let contentTopPaddingCompactForLargePhone: CGFloat = GlobalTokens.spacing(.sizeNone)
+        static let contentBottomPadding: CGFloat = GlobalTokens.spacing(.size80)
+        static let contentBottomPaddingCompact: CGFloat = GlobalTokens.spacing(.size80)
         static let accessoryHeight: CGFloat = 36
-        static let maxHeightNoAccessory: CGFloat = 56 - 44  // navigation bar - design: 56, system: 44
-        static let maxHeightNoAccessoryCompact: CGFloat = 44 - 32   // navigation bar - design: 44, system: 32
-        static let maxHeightNoAccessoryCompactForLargePhone: CGFloat = 44 - 44   // navigation bar - design: 44, system: 44
+        static let maxHeightNoAccessory: CGFloat = GlobalTokens.spacing(.size80)
+        static let maxHeightNoAccessoryCompact: CGFloat = GlobalTokens.spacing(.size80)
+        static let maxHeightNoAccessoryCompactForLargePhone: CGFloat = GlobalTokens.spacing(.sizeNone)
```

Comment:
Do we even need the `ForLargePhone` constants?

Reply:
Honestly, I don't think distinct `...CompactForLargePhone` constants have any business existing. Whether we use a `...Compact` constant or `...CompactForLargePhone` constant is determined by the *horizontal* size class, but the constants here talk about *vertical* spacing.

That being said, the `...CompactForLargePhone` constants (like on the Pro Max in the following screenshot) result in a more vertically centered title, so we might want to ultimately use those values. I'll play around with these some more and see what I can come up with.

<img width="660" alt="Screenshot 2023-04-21 at 11 26 00 AM" src="https://user-images.githubusercontent.com/717674/233718520-10f84796-f449-4854-b162-023269099544.png">

Reply:
Update on this: There actually *is* a good reason for the `...CompactForLargePhone` constants existing, and that's the fact that UIKit's default NavBar heights are indeed different for smaller phones (32px versus 44px). Whether or not we want to still maintain UIKit's default height is still up for discussion.

Reply:
We just confirmed that we want to do 44px everywhere, so I'm gonna mark this conversation as resolved for now and worry about consolidating these constants in another PR.